### PR TITLE
Add tag based CI/CD

### PR DESCRIPTION
**Pull request by cswendrowski**

### CODE REVIEW

File: .github/main.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          draft: false
+          prerelease: false
+          token: ${{ secrets.GITHUB_TOKEN }}
+          artifacts: './module.json,./furnace.zip'
+
+      # Update the 'latest' release
+      - name: Create Release
+        id: create_latest_release
+        uses: ncipollo/release-action@v1
+        if: endsWith(github.ref, 'master')
```
Comment (kakaroto) [inline .github/main.yml:51]: What is this for, if github.ref ends with 'master' ? would that fail due to the trigger being on tag push ?

**Comment by kakaroto** [comment]
Thanks CI Chief! Much appreciated! :)

I assume module.json would now need to be changed so it refers to `https://github.com/kakaroto/fvtt-module-furnace/releases/download/latest/module.json` and similarly for download url.

File: .github/main.yml (hunk 1 above)
Comment (cswendrowski) [reply .github/main.yml]: Good catch, I copied a line incorrectly. Removed!

**Comment by cswendrowski** [comment]
The `#Substitute the Manifest and Download URLs in the module.json` step will take care of that for the deployed asset, but I do recommend pointing your git submitted manifest at `https://github.com/kakaroto/fvtt-module-furnace/releases/download/latest/module.json` and `https://github.com/kakaroto/fvtt-module-furnace/releases/download/latest/furnace.zip`

**Comment by kakaroto** [comment]
Great, thanks for the quick fix! I'll test that soon 😁

**Comment by kakaroto** [comment]
😱 just realized the version in the module.json has the `v` so it changes teh `2.6.0` into `v2.6.0`

**Pull request merged**